.travis.yml

```diff
@@ -6,8 +6,9 @@ python:
 
 # command to install dependencies
 install:
+  - pip install yes
```

Comment:
Is `yes` actually required anywhere in the code? It's not used here in Travis. If it's needed in the code, then we should add it in requirement.txt

Reply:
Resolved in f53197e